.github/workflows/test.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         dbimage:
           - mysql:5.7
           - mysql:8.3
-          - mysql:9.2
+          - mysql:9.4
           - mariadb:10.4
           - mariadb:11
```

Comment:
We have mariadb 12 now, might want to add that?

Reply:
They finally released it, huh? Took long enough. But then, I care a lot less than I used to 😅